forum.js

```diff
@@ -0,0 +1,39 @@
+const express = require('express');
+
+module.exports = (rpc, conn, dbwrite) => {
+  const router = new express.Router();
+  router.get('/getAllForumPost', (req, res, next) => {
+    var rpcInput = {
+      method: 'allForumPost',
+      params: []
+    };
+    const channelName = 'forum_rpc_worker';
+    return rpc(conn, channelName, rpcInput)
+      .then(data => {
+        res.json(data);
+      })
+      .catch(next);
+  });
+
+  router.post('/createForumPost', (req, res, next) => {
+    var rpcInput = {
+      method: 'createForumPost',
+      params: [
+        {
+          auth0_id: req.body.auth0_id,
+          post_content: req.body.post_content,
+          stim_id: req.body.stim_id,
+          post_subject: req.body.post_subject,
+          created_at: req.body.created_at
+        }
+      ]
+    };
```

Comment:
fyi if you need to do this in the future, theres also `lodash#pluck` if you have lodash.

Reply:
seems like lodash removed _.pluck cause of the similarity between _.pluck and _.map
now we could use _.map(obj, key) to do the same thing, but acknowledged

Reply:
ah interesting, good catch